src/Compilers/CSharp/Portable/Syntax/LookupPosition.cs

```diff
@@ -289,7 +290,8 @@ private static SyntaxToken GetFirstIncludedToken(StatementSyntax statement)
                 case SyntaxKind.FixedStatement:
                     return ((FixedStatementSyntax)statement).FixedKeyword;
                 case SyntaxKind.ForEachStatement:
-                    return ((ForEachStatementSyntax)statement).OpenParenToken.GetNextToken();
+                case SyntaxKind.ForEachComponentStatement:
```

Comment:
No, but I will for the next revision.

---
In reply to: [73614661](https://github.com/dotnet/roslyn/pull/12917#discussion_r73614661) [](ancestors = 73614661)